src/components/controls/TransitToggleButtonBar.tsx

```diff
@@ -6,36 +6,42 @@ import * as s from './transitToggleButtonBar.scss';
 
 interface ITtransitToggleButtonBarProps {
     selectedTransitTypes: TransitType[];
-    toggleSelectedTransitType: (type: TransitType) => void;
+    toggleSelectedTransitType?: (type: TransitType) => void;
```

Comment:
Set this to nullable, in order for us to also use this bar in static environments.